### PR TITLE
Selenium testing

### PR DESCRIPTION
**Pull request by toban**
- adds integration test between wikibase, queryservice and queryservice-ui
- add makefile
- update readme
- test step uploads testing artifacts

### CODE REVIEW

File: test_scripts/selenium/wdio.conf.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+	// Level of verbosity: silent | verbose | command | data | result | error
+	logLevel: 'error',
+
+	// Setting this enables automatic screenshots for when a browser command fails
+	// It is also used by afterTest for capturig failed assertions.
+	screenshotPath: process.env.LOG_DIR || __dirname + '/log',
+
+	// Default timeout for each waitFor* command.
+	waitforTimeout: 20 * 1000,
```
Comment (toban) [inline test_scripts/selenium/wdio.conf.js:66]: this was bumped to 20 seconds

File: .github/workflows/built_and_test.yml [hunk 2]
```diff
@@ -134,3 +134,12 @@ jobs:
 
       - name: Run Docker Tests
         run: bash test_scripts/test_docker.sh
+
+      - name: Archive docker test artifacts
+        uses: actions/upload-artifact@v2
+        if: always()
```
Comment (mariushoch) [inline .github/workflows/built_and_test.yml:140]: Why is that needed?
